services/ux-backend/handlers/onboarding_tokens.go

```diff
@@ -0,0 +1,110 @@
+package handler
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/base64"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/red-hat-storage/ocs-operator/v4/services/types"
+)
+
+func OnboardingTokensHandler(w http.ResponseWriter, r *http.Request) {
+
+	switch r.Method {
+	case "POST":
+		onboardingToken, err := getOnboardingToken()
+		if err != nil {
+			http.Error(w, encodeResponse(types.HTTPResponse{Code: http.StatusMethodNotAllowed, Message: onboardingToken}), http.StatusInternalServerError)
+			return
+		}
+
+		onboarding := struct {
+			Token string `json:"token"`
+		}{}
+
+		onboarding.Token = onboardingToken
```

Comment:
why can't, pls do format it properly though.
```suggestion
		onboarding := struct {Token string `json:"token"`}{Token: onboardingToken}
```